projects/plugins/jetpack/extensions/blocks/videopress/edit.js

```diff
@@ -780,12 +830,15 @@ export default createHigherOrderComponent(
 			const preview = !! url && getEmbedPreview( url );
 
 			const isFetchingEmbedPreview = !! url && isRequestingEmbedPreview( url );
-			const isUploading = isBlobURL( src );
+			const resumableUploadEnabled = typeof window.videoPressResumableEnabled !== 'undefined';
```

Comment:
The variable name "videoPressResumableEnabled" suggests it's a boolean, so it could exist in a false state, should this be :
```suggestion
			const resumableUploadEnabled = typeof window.videoPressResumableEnabled !== 'undefined' && window.videoPressResumableEnabled;
```
or even
```suggestion
			const resumableUploadEnabled = !! window.videoPressResumableEnabled;
```
?

Reply:
Prefer the second one!